src/napari_imagej/widgets/resources/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""
+A module used to help find napari-imagej widget resources
```

Comment:
The idea of resources is more general than only for widgets. Consider moving this nice utility class up one level, along with all the icons.

Reply:
Moved up in 4e5163e